src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Logger/Logger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Logger
+{
+    public class Logger
+    {
+        private static Action<string> WriteWarningAction { get; set; }
```

Comment:
Is there a need for them to be properties? Can them being fields work just fine?